### PR TITLE
feature flag relaychain runtimes

### LINKED ISSUE #175: Building the Collator builds all the Polkadot runtimes.
When building the test collator with `cargo build --release -p rococo-validator`, all polkadot runtimes are built.  For example it builds `kusama-runtime`, `westend-runtime`, `polkadot-test-runtime`, etc.

This is harmless in terms of correctness, but it adds a lot of build time.

### PR DESCRIPTION
**Pull request by xlc**
Closes #175

The compiling time of building 5 runtimes (4 relaychain + 1 parachain, maybe twice wasm & native?) is literally killing my life.

Also relay chain runtime std feature is currently not enabled even in std because cargo. The dependent package can however depends on runtime directly and enable it.

### CODE REVIEW

**Comment by xlc** [comment]
@bkchr does it currently builds wasm file as well? Is there a way to disable wasm building?

File: upward-message/Cargo.toml [hunk 1]
```diff
@@ -13,30 +13,27 @@ polkadot-core-primitives = { git = "https://github.com/paritytech/polkadot", bra
...
 std = [
 	"sp-std/std",
-	"polkadot-runtime/std",
-	"kusama-runtime/std",
-	"westend-runtime/std",
-	"rococo-runtime/std",
 	"polkadot-core-primitives/std",
 	"polkadot-parachain/std",
 ]
-# Should be enabled by when compiling the runtime for WASM.
```
Comment (bkchr) [inline upward-message/Cargo.toml:32]: Please add back.

**Comment by xlc** [comment]
The feature handling with cargo makes thing bit more complicated than what I was expecting... Does it make sense for me to continue on this PR or just close it because it may not actually solve my problem?

**Comment by bkchr** [comment]
> The feature handling with cargo makes thing bit more complicated than what I was expecting... Does it make sense for me to continue on this PR or just close it because it may not actually solve my problem?

As you like, but to solve this correctly we will also need to make the runtimes optional to the polkadot-service.

**Comment by xlc** [comment]
Ok. Since this doesn't look like a low hanging fruit anymore, I will close this.

**Pull request closed without merging**